api/client/clientRouter.js

```diff
@@ -7,7 +7,7 @@ const Ethnicity = require('../ethnicities/ethnicitiesModel');
 const EducationHistory = require('../education_histories/educationHistoriesModel');
 const EmailAddress = require('../email_addresses/emailAddressesModel');
 const PhoneNumber = require('../phone_numbers/phoneNumbersModel');
-
+const PlaceHolder = require('./clientModel');
```

Comment:
Hi @Smarsden007 : wondering why this is needed since you already have a 'Clients' var accessing the same model.